app/poros/score.rb

```diff
@@ -0,0 +1,8 @@
+class Score
```

Comment:
I'm wondering if we should make this a model class rather than a PORO. I'm thinking if we have multiple moderators accessing this information, it will make separate calls to the `PerspectiveService` rather than one call that gets saved to the DB. 

We could also just make it an attribute of a squeak since the only metric we are using is `:IDENTITY_ATTACK` and the only value associated is the probability.

Reply:
I was considering whether it should be saved in the db or invoked on fetch with most up-to-date scoring since the scoring is supposed to continuously evolve. Someone had mentioned that it would be nice to include :TOXICITY as well. I was hoping to consult Dara on best practices too.

Reply:
That's a good point about being extensible to other measures, I'm going to merge as is and we can look at caching if we want to reduce calls